deploy/action.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy to CloudHub
+description: Deploy to CloudHub with Maven
+inputs:
+  use_artifact:
+    description: Download artifact
+    default: 'true'
```

Comment:
as it's an option, the user can set it to `false` right @hoangmirs and with that, the whole workflow will be failed, is it, for that reason, I think is is a mandatory field actually, what do you think?

Reply:
IMO, it could be an optional field, just make sure that the value here and the `build` workflow should be the same. If use don't use_artifacts (GitHub artifacts), they still use both within a job.